netty/src/main/java/io/grpc/netty/ProtocolNegotiators.java

```diff
@@ -363,13 +415,23 @@ static HostPort parseAuthority(String authority) {
     return new HostPort(host, port);
   }
 
+  /**
+   * Returns a {@link ProtocolNegotiator} that ensures the pipeline is set up so that TLS will
+   * be negotiated, the {@code handler} is added and writes to the {@link io.netty.channel.Channel}
+   * may happen immediately, even before the TLS Handshake is complete.
```

Comment:
Do you want to mention the additional param? Otherwise comment is identical to the next one.